### PR TITLE
main/kamailio: Updated package version. Added kazoo module.

### PR DESCRIPTION
**Pull request by sergey-safarov**

### CODE REVIEW

File: main/kamailio/APKBUILD [hunk 1]
```diff
@@ -275,7 +272,7 @@ prepare() {
 
 build() {
 	cd "$_builddir"
-	make FLAVOUR=kamailio STUN=1 \
+	make -j16 FLAVOUR=kamailio STUN=1 \
```
Comment (fabled) [inline main/kamailio/APKBUILD]: Is this needed? Normally abuild.conf sets JOBS and MAKEFLAGS to indicate proper parallelism level.

File: main/kamailio/APKBUILD [hunk 2]
```diff
@@ -229,20 +229,17 @@ for _i in db postgres sqlite dbtext mysql \
...
    eval "_modules=\"\$_modules \$_mod_list_$_i\""
 done
 
 source="http://www.kamailio.org/pub/kamailio/$pkgver/src/${pkgname}-${pkgver}${_suffix}.tar.gz
-	0001-musl-fixes.patch
-	0002-fix-libressl.patch
-	0003-kamdbctl.base.patch
-	0004-remove-spurious-execinfo.patch
+	0001-kamdbctl.base.patch
+	0002-remove-spurious-execinfo.patch
```
Comment (fabled) [inline main/kamailio/APKBUILD]: I have not usually renamed patches in situation like this, but I guess it's ok to do so if wanted.

Comment (sergey-safarov) [inline main/kamailio/APKBUILD]: As i understand patches applied in alphabetical order.
I ordered patches to prevent wrong patch order with future patches.

Comment (fabled) [inline main/kamailio/APKBUILD]: Patches are applied in the order listed in `source`.

Comment (sergey-safarov) [inline main/kamailio/APKBUILD]: Is still required revert back file names?

Comment (fabled) [inline main/kamailio/APKBUILD]: Not required. Just a comment that changing them is not required either :)

Comment (sergey-safarov) [inline main/kamailio/APKBUILD]: Please delete change request.
PR marked as not ready to merge.

**Comment by fabled** [comment]
Committed with two minor changes (reset pkgrel=0 and mentioned version in commit message). Thanks.

**Pull request closed without merging**